packages/frontend/package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "clean": "rimraf ./dist",
     "build": "npm run clean && cross-env BUILD_PATH='./dist' react-scripts build",
-    "start:dev": "cross-env BROWSER=none PORT=3001 BUILD_PATH='./dist' react-scripts start"
+    "start:dev": "eslint ./src --fix && cross-env BROWSER=none PORT=3001 BUILD_PATH='./dist' react-scripts start"
```

Comment:
@stemmlerjs, why do you need `eslint ./src --fix` here? The code must have been linted by the time someone ever starts writing new code.